server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -243,9 +263,9 @@ func (r *Runner) Run(ctx workflow.Context) error {
 		return r.toExternalError(err, "fetching root")
 	}
 	defer func() {
-		err := cleanup()
+		cleanupErr := cleanup()
 
-		if err != nil {
+		if cleanupErr != nil {
 			logger.Warn(ctx, "error cleaning up local root", "err", err)
```

Comment:
nit: should this be `cleanupErr`?  
```suggestion
			logger.Warn(ctx, "error cleaning up local root", "err", cleanupErr)
```